staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -394,7 +394,7 @@ func (_ Quantity) OpenAPISchemaType() []string { return []string{"string"} }
 
 // OpenAPISchemaFormat is used by the kube-openapi generator when constructing
 // the OpenAPI spec of this type.
-func (_ Quantity) OpenAPISchemaFormat() string { return "" }
+func (_ Quantity) OpenAPISchemaFormat() string { return "int-or-string" }
```

Comment:
we don't support this format.

And the correct schema would be `x-kubernetes-int-or-string-or-float: true`, and we don't have that.